scripts/content.js

```diff
@@ -58,5 +58,5 @@ const setMutationObserver = (callback) => {
   // Start observing the target node for configured mutations
   observer.observe(targetNode, config);
 }
-
-setMutationObserver(callback);
+addBorders();
```

Comment:
I'll have to deploy this change after the review process completes.

Reply:
(The permissions update is included in the initial Chrome extension submission.)